homeassistant/components/asuswrt/bridge.py

```diff
@@ -73,20 +72,17 @@ def _handle_errors_and_zip(func: _FuncType) -> _ReturnFuncType:
 
         @functools.wraps(func)
         async def _wrapper(self: _AsusWrtBridgeT) -> dict[str, Any]:
-            try:
-                data = await func(self)
-            except exceptions as exc:
```

Comment:
Handling exceptions is one of the points of this decorator. If we remove that we need to redesign how the decorator is used and defined.